src/Microsoft.VisualStudio.Web.CodeGeneration.EntityFrameworkCore/Microsoft.VisualStudio.Web.CodeGeneration.EntityFrameworkCore.csproj

```diff
@@ -17,7 +17,7 @@
        without adding EntityFrameworkCore dependency.
        https://github.com/aspnet/scaffolding/issues/434
        -->
-    <PackageReference Include="Microsoft.EntityFrameworkCore.Design" Version="$(AspNetCoreVersion)" PrivateAssets="All" />
+    <PackageReference Include="Microsoft.EntityFrameworkCore.Design" Version="$(AspNetCore201PatchVersion)" PrivateAssets="All" />
```

Comment:
This should already be covered by https://github.com/aspnet/Scaffolding/blob/rel/2.0.1/build/dependencies.targets. It's a little hacky for now, but we want to eventually move away from having lots of variables in dependencies.props and using instead a list that has an exact `packageID => version` mapping